### PR TITLE
Adds builder call for Dante's greeting and a print statement

### PR DESCRIPTION
**Pull request by DanteMillerDS**
Adds builder for dantes greeting. Prints a statement.

### CODE REVIEW

**Review by JadynSondrol — DISMISSED** [review]
Looks good!

**Review by Jeklund99 — DISMISSED** [review]

**Review by JadynSondrol — DISMISSED** [review]
Looks good!

**Review by jquist24 — DISMISSED** [review]
Looks good. Correct alphabetization.

**Review by Jeklund99 — DISMISSED** [review]
Looks good.

**Review by ellst029 — DISMISSED** [review]

**Review by dahlgrenthomas — DISMISSED** [review]
Looks good.

**Review by IsabelleHjelden — DISMISSED** [review]

**Review by JadynSondrol — DISMISSED** [review]

**Review by jquist24 — DISMISSED** [review]
Good.

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -22,8 +22,14 @@ public String generateOutput() {
 
     
     builder.append(cassieSaysHello());
+    builder.append(DanteSaysHello());
+
+
+
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: You probably don't want all this blank space.
```suggestion
```

**Review by brwjxxvi — DISMISSED** [review]
Looks good

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (NicMcPhee) [reply src/main/java/hellos/Hellos.java]: And same for the blank lines below.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -42,6 +48,9 @@ private String nicSaysHello() {
   private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
+  private String DanteSaysHello() {
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:43]: I _would_ put a blank link between method definitions, though.

```suggestion

  private String DanteSaysHello() {
```

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
I would make these changes before you actually merge into `main`.

**Review by JadynSondrol — DISMISSED** [review]

**Review by RichardLussier — DISMISSED** [review]
This is in proper order

**Review by LloydHilsgen001 — DISMISSED** [review]
Looks good!

**Review by DanteMillerDS — COMMENTED** [review]
got rid of the spaces

**Review by DanteMillerDS — COMMENTED** [review]
made changes

**Review by PhallyPhal — DISMISSED** [review]
Looks good.

**Review by morrisschooljm17 — APPROVED** [review]
Looks good to me!

**Review by NicMcPhee — APPROVED** [review]
This looks great!

**Pull request merged**